### DISCUSSION TITLE
ModuleNotFoundError: No module named 'chromadb' even after pip install chromadb

### DISCUSSION THREAD

**Discussion by Mayurk619** [imartinez/privateGPT#975] [Q&A]
I'm installing private GPT and ran ingest.py and I got this error `ModuleNotFoundError: No module named 'chromadb'`. Kindly help

**Comment by Jawn78** [comment]
You need to install chromadb. In my case I installed it manually, but for whatever reason I needed to install the requirements.txt again after running

`pip install chromadb`
